specification/cognitiveservices/data-plane/WebSearch/v1.0/WebSearch.json

```diff
@@ -0,0 +1,1211 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Web Search API",
+    "description": "The Web Search API lets you send a search query to Bing and get back search results that include links to webpages, images, and more.",
+    "version": "1.0"
+  },
+  "host": "api.cognitive.microsoft.com",
+  "schemes": [
+    "https"
+  ],
+  "basePath": "/bing/v7.0",
+  "produces": [
```

Comment:
We should probably have a `consumes` as well.

Reply:
We don't really consume anything, we only have a GET method

Reply:
lol.

WHOOSH! 

No worries. 😄